hack/generator/pkg/astmodel/object_type.go

```diff
@@ -50,7 +51,7 @@ func (objectType *ObjectType) AsDeclarations(codeGenerationContext *CodeGenerati
 		},
 	}
 
-	addWrappedComments(&declaration.Doc.List, description, 200)
+	astbuilder.AddWrappedComments(&declaration.Doc.List, description, 200)
```

Comment:
You could use the `StringLiteral` here in the `defineField` function.

Reply:
Used `LiteralString()` to simplify as suggested.

Reply:
Followup - renamed to `StringLiteral()` and introduced `TextLiteral()` because some usage doesn't want `"`quotes`"`.